homeassistant/components/plugwise/const.py

```diff
@@ -22,7 +22,6 @@
 DEFAULT_TIMEOUT = 60
 
 # Configuration directives
-CONF_BASE = "base"
```

Comment:
Why are you removing this one? It is used in `config_flow.py`.
And, if you are planning to import it from Core `const.py`, then the import is missing.

Reply:
It is already defined in `homeassistant/const.py`. If this was actually needed, pylint, flake8, or the test suite would have failed this PR. This duplicate definition was not detected by tests because `CONF_BASE` is being used and the `config_flow.py` already imported the `homeassistant/const.py` definition.

Reply:
Ok, yes, I see we are already doing that.
The title of this PR is confusing, the integration is already using core constants. You are proposing to do little cleaning-up.

No problem, approved.